nifi-testharness/src/test/java/org/apache/nifi/testharness/samples/TestUtils.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+
+package org.apache.nifi.testharness.samples;
+
+import java.io.File;
+
+final class TestUtils {
+
+    private TestUtils() {
+        // no instances allowed
+    }
+
+    static File getBinaryDistributionZipFile(File binaryDistributionZipDir) {
+
+        File[] files = binaryDistributionZipDir.listFiles((dir, name) ->
```

Comment:
This can return `null` which makes the next `if` statement fail with a NPE. I think the condition can be extended to

```
if (files == null || files.length == 0)
```